domains/test/service/src/domain.rs

```diff
@@ -456,6 +471,13 @@ where
             ReputationChange::new(i32::MAX, "Peer unbanned by test (2)"),
         );
     }
+
+    /// Take and stop the domain node and delete its database lock file
+    pub fn stop(self) -> Result<(), std::io::Error> {
+        // Remove the database lock file
+        std::fs::remove_file(self.base_path.path().join("paritydb").join("lock"))?;
```

Comment:
On Windows, sometimesan open file can’t be deleted. So I suggest making two changes:
1. drop first
2. print failures and do not error

```suggestion
        // On Windows, sometimes open files can’t be deleted
        std::mem::drop(self);
        // The lock file already being deleted is not a fatal test error, so just log it
        let result = std::fs::remove_file(self.base_path.path().join("paritydb").join("lock"));
        if let Err(e) = result {
            println!("deleting paritydb lock file failed: {e:?}");
        }
```